py/server/tests/test_parquet.py

```diff
@@ -570,19 +570,16 @@ def test_read_parquet_from_s3(self):
 
         # Fails since we have a negative read_ahead_count
         with self.assertRaises(DHError):
-            s3.S3Instructions(region_name="us-east-1",
-                              read_ahead_count=-1,
-                              )
+            s3.S3Instructions(read_ahead_count=-1)
 
         # Fails since we provide the key without the secret key
         with self.assertRaises(DHError):
             s3.S3Instructions(region_name="us-east-1",
                               access_key_id="Some key without secret",
                               )
 
-        s3_instructions = s3.S3Instructions(region_name="us-east-1",
-                                            read_ahead_count=1,
-                                            )
+        s3_instructions = s3.S3Instructions()
```

Comment:
We should probably have a test where we don't pass `special_instructions` in too?